### PR TITLE
docs/unified/novacustom/hcl.md: update

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

File: docs/unified/novacustom/hcl.md [hunk 1]
```diff
@@ -130,6 +130,17 @@ issue from which the report is sourced.
         1. Rev.C1 and Rev.E
         2. DP Alt mode: Synaptics VMM5310 DP MST hub, two upstream DP 1.4 lanes,
            DSC 1.2 decompression, up to 2x 4K60 + 1x 4K30 depending on source.
+    === "V540TU MTL Ultra 5 14th Gen."
```
Comment (mkopec) [inline docs/unified/novacustom/hcl.md:133]:
```suggestion
    === "V54 / V56 14th Gen."
```

All MTL models (dGPU/iGPU, 14 and 16 inch) will be tested with that dock

Comment (wiktormowinski) [reply docs/unified/novacustom/hcl.md:133]: ok, other than that the syntax is correct? I believe i have no way to preview it, i might be wrong tho.

Comment (mkopec) [reply docs/unified/novacustom/hcl.md]: Syntax looks good. In the future if you want to preview locally you can follow https://github.com/Dasharo/docs?tab=readme-ov-file#local-build

**Review by mkopec — APPROVED** [review]

File: docs/unified/novacustom/hcl.md (hunk 1 above)
Comment (wiktormowinski) [reply docs/unified/novacustom/hcl.md]: thanks, i will use that in the future for sure

**Pull request merged**